crates/libtiny_tui/src/tui.rs

```diff
@@ -90,6 +93,12 @@ pub struct TUI {
     config_path: Option<PathBuf>,
 }
 
+pub(crate) enum CmdResult {
+    Ok,
+    Skip,
+    Quit,
+}
```

Comment:
Would be good to document these variants. As far as I understand:

- `Ok`: command is handled by the TUI, don't propagate it to the caller.
- `Skip`: Command is not handled by the TUI, propagate it to the caller.
- `Quit`: Quit requested

If you document it this way then it becomes clear that "skip" is a bit confusing. It feels like the receiver (whoever got the `Skip` value) should skip, but actually it means the sender (whoever created the `Skip` value) is skipped so the receiver should handle it, right? Perhaps it should be called `Ignored` or `Skipped`.

Reply:
Changed to `Continue`